### PR TITLE
Update searching-for-repositories.md

### LINKED ISSUE #17388: GitHub Search API Irregularity
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/search-github/searching-on-github/searching-for-repositories

### What part(s) of the article would you like to see updated?

Remove the part in the linked article where it specifies that "**When you omit this qualifier, only the repository name and description are searched**". This is confirmed to be untrue since for example:
curl -s "https://api.github.com/search/repositories?q=graphql"
curl -s "https://api.github.com/search/repositories?q=graphql+in:name,description"
Yields different results.

A GitHub Support also confirmed that this is the case, at lest at the moment (presumably going forward as well).

The GitHub Support representative did not specify what fields were searched, just that it wasn't only name and description.
My own guess is that topics where also searched, but I am unable to fully confirm this.

TLDR; GitHub docks 'promises' behavior that is not followed. 

### Additional information

See: https://github.com/github/docs/discussions/17192 for more details

### PR DESCRIPTION
**Pull request by gusbo010**
### Why:

Closes #17388 
Discussion #17192

### What's being changed:
content/search-github/searching-on-github/searching-for-repositories.md
Remove: **When you omit this qualifier, only the repository name and description are searched.**

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@gusbo010 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by kag728** [comment]
Hi @gusbo010, thanks for opening this PR! It looks like you're right, when we don't specify an `in` qualifier, we default to search repository name, description and topics. Instead of removing the entire line, can we add in that topics are searched too? I have suggested the change on the commit. Thanks again!

File: content/search-github/searching-on-github/searching-for-repositories.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ To include forks in the search results, you will need to add `fork:true` or `for
 
 ## Search by repository name, description, or contents of the README file
 
-With the `in` qualifier you can restrict your search to the repository name, repository description, contents of the README file, or any combination of these. When you omit this qualifier, only the repository name and description are searched.
+With the `in` qualifier you can restrict your search to the repository name, repository description, contents of the README file, or any combination of these. When you omit this qualifier, only the repository name, description, and topics are searched.
```
Comment (kag728) [inline content/search-github/searching-on-github/searching-for-repositories.md:26]:
```suggestion
With the `in` qualifier you can restrict your search to the repository name, repository description, contents of the README file, topics, or any combination of these. When you omit this qualifier, only the repository name, description, and topics are searched.
```

**Comment by kag728** [comment]
Hey @gusbo010! Thanks for waiting. This feature is fully rolled out. Before we merge this docs change, can we add `in:topics` as an example in the table below where the changes we've already made are? Once we add an example in, we'll be good to go here! Thanks 😄

**Review by skedwards88 — COMMENTED** [review]
Thanks for this addition @gusbo010 ! Can you add an example to the table (lines 28-35) for `in:topic` (based on [this comment](https://github.com/github/docs/pull/17389#issuecomment-1156618517))? Once that is added, we'll merge this down for you.

**Comment by gusbo010** [comment]
@kag728, @skedwards88.
Example now added. The example I added is written using the assumption that in:topic matches exactly to specified query.
I.e. "java" matches topic "java" not topic "javascript". If this is not the case it should be reworded to something else.

**Review by skedwards88 — APPROVED** [review]
✨ Thank you! We'll get this merged down for you.

**Pull request merged**